bot/exts/halloween/spookygif.py

```diff
@@ -1,38 +1,38 @@
 import logging
 
-import aiohttp
 import discord
 from discord.ext import commands
 
+from bot.bot import Bot
 from bot.constants import Tokens
 
 log = logging.getLogger(__name__)
 
+API_URL = "http://api.giphy.com/v1/gifs/random"
+
 
 class SpookyGif(commands.Cog):
     """A cog to fetch a random spooky gif from the web!"""
 
-    def __init__(self, bot: commands.Bot):
+    def __init__(self, bot: Bot):
         self.bot = bot
 
     @commands.command(name="spookygif", aliases=("sgif", "scarygif"))
     async def spookygif(self, ctx: commands.Context) -> None:
         """Fetches a random gif from the GIPHY API and responds with it."""
         async with ctx.typing():
-            async with aiohttp.ClientSession() as session:
-                params = {'api_key': Tokens.giphy, 'tag': 'halloween', 'rating': 'g'}
-                # Make a GET request to the Giphy API to get a random halloween gif.
-                async with session.get('http://api.giphy.com/v1/gifs/random', params=params) as resp:
-                    data = await resp.json()
-                url = data['data']['image_url']
+            params = {"api_key": Tokens.giphy, "tag": "halloween", "rating": "g"}
+            # Make a GET request to the Giphy API to get a random halloween gif.
+            async with self.bot.http_session.get(API_URL, params=params) as resp:
+                data = await resp.json()
+            url = data["data"]["image_url"]
 
-                embed = discord.Embed(colour=0x9b59b6)
-                embed.title = "A spooooky gif!"
-                embed.set_image(url=url)
+            embed = discord.Embed(title="A spooooky gif!", colour=0x9b59b6)
```

Comment:
Other files have the colour hex as a constant. We shall keep it consistent. Here is an example of a colour existing as a constant as proof: https://github.com/python-discord/sir-lancebot/blob/92c33d625068b9b39564dbf4fb9f6c1102711955/bot/exts/evergreen/snakes/_snakes_cog.py#L32